content/en/docs/concepts/workloads/autoscaling.md

```diff
@@ -0,0 +1,55 @@
+---
+reviewers: []
+title: Autoscaling Workloads
+description: >-
+  With Autoscaling, you can automatically update your workloads or infrastructure in one way or another. This allows your cluster to react to changes in resource demand more elastically and efficiently.
+content_type: concept
+weight: 40
+hide_summary: true # Listed separately in section index
+---
+
+<!-- overview -->
+
+The concept of _Autoscaling_ in Kubernetes refers to the ability to automatically update the available
+resources of your cluster. This can be either a replication controller (for example a
+{{< glossary_tooltip text="Deployment" term_id="deployment" >}} or
+{{< glossary_tooltip text="ReplicaSet" term_id="replica-set" >}}), or the cluster infrastructure
+itself (for example the number of {{< glossary_tooltip text="Nodes" term_id="node" >}}).
+
+Besides the differentiation in _what_ is being scaled, there are also different options for _how_ to scale:
+
+- scale the number of available instances (such as Pods or Nodes)
+- scale the available resources on existing instances themselves (such as CPU or memory)
+
+The first option is referred to as _horizontal scaling_, while the second is referred to as _vertical scaling_.
+
+<!-- body -->
+
+## Scaling Workloads Horizontally
```

Comment:
We write headings (not page titles, but headings) in [sentence case](https://kubernetes.io/docs/contribute/style/style-guide/#headings)